scripts/split-paired-reads.py

```diff
@@ -92,11 +92,15 @@ def main():
 
     infile = args.infile
 
-    check_input_files(infile, args.force)
```

Comment:
This line needs to be added back in for situations where filenames are provided.